easybuild/easyconfigs/g/gnuplot/gnuplot-5.2.8-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = 'ConfigureMake'
+
+name = 'gnuplot'
+version = '5.2.8'
+
+homepage = 'https://gnuplot.sourceforge.net/'
+description = """Portable interactive, function plotting utility"""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+
+source_urls = [('https://sourceforge.net/projects/gnuplot/files/gnuplot/%(version)s', 'download')]
+sources = [SOURCE_TAR_GZ]
+patches = ['%(name)s-%(version)s.patch']
+checksums = [
+    '60a6764ccf404a1668c140f11cc1f699290ab70daa1151bb58fed6139a28ac37', # gnuplot-5.2.8.tar.gz
+    'bd224d6ce4520259a83b3d1bd7eb43dc3e0af72c786af30114a89594c83d7f30', # gnuplot-5.2.8.patchy
```

Comment:
It doesn't really matter since this is a comment, but if it should match the filename, fix typo patchy.